### ISSUE TITLE
Update the `Finding and Installing a Plugin` docs

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/docs#122]
## 🧰 Task

napari PR https://github.com/napari/napari/pull/5198 overhauled the plugin installation GUI, implementing a new Plugin Manager. As a result, the existing docs:
https://napari.org/stable/plugins/find_and_install_plugin.html#finding-and-installing-a-napari-plugin
need to be updated.
CC: @ppwadhwa @goanpeca

**Comment by melissawm** [comment]
I believe this is completed, can you confirm @psobolewskiPhD ?

**Comment by psobolewskiPhD** [comment]
I don't think so https://napari.org/stable/plugins/start_using_plugins/finding_and_installing_plugins.html#find-and-install-plugins still doesn't reflect the current state of the plugin manager -- which is soon to change, in that the plugins will not be listed by default but require filtering (ala VS Code extensions).
Importantly, it doesn't mention the possibility of using conda vs pip.
We can wait to update once the plugin manager UI is settled?


Edit: actually, the plugin manager has nice docs now:
https://napari.org/napari-plugin-manager/

we could link to those?

**Comment by melissawm** [comment]
Can you clarify - do you mean we can install a plugin from within the napari plugin manager using conda, or that you can use conda-forge to install a plugin?

**Comment by psobolewskiPhD** [comment]
Both? 🤣

**Comment by melissawm** [comment]
Sorry, I don't follow. If I type a name in the search bar, it's installed with pip, right? (even if I'm inside a conda env)

I guess my question is what information is missing from the guide (and my PR update?)

**Comment by psobolewskiPhD** [comment]
The big change:
- `Installation Info` allows you to pick conda(forge) or pip, defaulting to how napari was installed. In the bundle, currently it's just conda, but with the next release will allow over-ridding to install from pip.
- the bottom text entry bar also allows you to choose conda or pip. This bar is actually quite powerful: it can install arbitrary packages.

Merged, but not released:
- UI/UX: you need to start typing to search -- there is no longer a pre-populated list.

The docs are hosted on napari.org maybe we can just integrate them?
https://napari.org/napari-plugin-manager/

**Comment by melissawm** [comment]
Can you give me an example of how to choose conda with the bottom text entry?

**Comment by psobolewskiPhD** [comment]
It's shown here:
https://napari.org/napari-plugin-manager/#installing-a-plugin-via-direct-entry
But there is a catch: apparently conda only shows if you installed napari from conda!
I was very confused because my dev env didn't show it (pip -e install) while my 0.5.5 env did (conda install).
I think the reason for this is that conda doesn't check site-packages so it will reinstall everything and good chance of breaking the env. Pip does check site-packages so it shouldn't reinstall much.
So those plugin manager docs should make that clearer.

**Comment by melissawm** [comment]
Got it! Thanks - I'll try to word that correctly in our napari pages.

**Comment by melissawm** [comment]
Wow that is powerful indeed - just added a commit to address it in https://github.com/napari/docs/pull/541#issue-comment-box
